### DISCUSSION TITLE
Netlify CMS -> Static CMS

### DISCUSSION THREAD

**Discussion by simplerethink** [lumeland/lume#354] [General]
Based on [this discussion](https://answers.netlify.com/t/is-this-project-dead/70988/61) it seems like Netlify CMS might be discontinued.

There is a community fork [here that appears to be active](https://github.com/StaticJsCMS/static-cms). 

Not sure it has any meaningful impact to you at this point, but just an FYI, since I see Netlify CMS mentioned throughout your project.

**Comment by oscarotero** [comment]
Oh, I didn't know that. Thank you for your notice. Probably I'll switch to Static CMS when the version 1.0 is stabilized.

**Comment by oscarotero** [comment]
Looks like Netlify CMS was picked by other company and renamed to Decap CMS https://decapcms.org/
